src/package-vsix.js

```diff
@@ -106,19 +108,32 @@ let version = '1.39.1-prel';
         } finally {
             fs.writeFileSync(pckPath, originalContent, 'utf-8');
             fs.removeSync(readmePath);
+            fs.removeSync(extLicense);
         }
-
     }
-    
+
     console.log(result.join(os.EOL));
 })();
 
 // a very basic README to add to the extension to explain what it is
 function genReadme(ext) {
     return `# Built-in extension: ${ext}
-        
-Built-in are extensions that are included in \`VS Code\` and \`Code OSS\` They are part of the [vscode GitHub repository](https://github.com/microsoft/vscode/tree/master/extensions) and built along with it. 
 
-So if you are running \`VS Code\` or \`Code OSS\` you probably do not need to add them, but other editors might`;
+## Disclaimer
     
+Microsoft does not endorse, build, test or publish this extension, nor are they involved with it in any other way. The only association is that they own the copyright of these extensions and the rest of vscode's [source code](https://github.com/microsoft/vscode/tree/master/extensions), which they released under the [MIT License](https://github.com/microsoft/vscode/blob/master/LICENSE.txt). A copy of the license is included in this extension. See LICENSE-vscode.txt
+
+    "Original VS Code sources are Copyright (c) 2015 - present Microsoft Corporation."
+
+
+## What is this extension? Do I need it?
+
+TL;DR: If you are running \`VS Code\`, \`Code OSS\` or derived product built from the VS Code repository, such as [VSCodium](https://github.com/VSCodium/vscodium), you do not need to install this extension since it's already present - "builtin".
```

Comment:
```suggestion
TL;DR: If you are running \`VS Code\`, \`Code OSS\` or a derived product built from the VS Code repository, such as [VSCodium](https://github.com/VSCodium/vscodium), you do not need to install this extension since it's already present - "built-in".
```

Reply:
done